server/cluster/cluster.go

```diff
@@ -53,6 +54,7 @@ import (
 )
 
 var backgroundJobInterval = 10 * time.Second
+var saveMinResolvedTSInterval = 1 * time.Second
```

Comment:
Do we need to make it configurable?

Reply:
cc @disksing @nolouch

Reply:
I think we need a configuration. maybe we should make it disabled by default.

Reply:
ok